vanilla/scss/styles.scss

```diff
@@ -15,23 +15,74 @@ $grid-max-width: 92rem;
   color: $colors--light-theme--text-default;
 }
 
-//The following is a workaround as after spending a while exploring I can't work out how to escape the -- in the class name from the .rst file
-.p-notification-information {
-  @extend .p-notification--information;
+li > p {
+  margin-bottom: 0;
 }
 
-.p-notification-negative {
-  @extend .p-notification--negative;
-}
+// Reuses style from p-notification pattern
+// https://github.com/canonical-web-and-design/vanilla-framework/blob/main/scss/_patterns_notifications.scss#L21
+.admonition {
+  @extend %vf-is-bordered;
+  @extend %vf-bg--x-light;
 
-.p-notification-caution {
-  @extend .p-notification--caution;
-}
+  background-position: $sph-inner $notification-icon-vert-offset;
+  background-repeat: no-repeat;
+  background-size: map-get($icon-sizes, default);
+  border-radius: 0 $border-radius $border-radius 0;
+  margin-bottom: $spv-outer--scaleable;
+  padding-bottom: calc(#{$spv-inner--small} - 1px);
+  padding-left: $notification-content-icon-space;
+  padding-top: calc(#{$spv-inner--small} - 1px);
+  position: relative;
 
-.p-notification-positive {
-  @extend .p-notification--positive;
-}
+  &::before {
+    height: auto;
+    width: 0.1875rem;
+    bottom: -1px;
+    top: -1px;
+    z-index: 1;
+    content: "";
+    position: absolute;
+    left: 0;
+  }
+  // This is reducing the bottom margin on the content text with no class name
+  > p {
+    margin-bottom: $notification-text-margin-bottom;
+  }
 
-li > p {
-  margin-bottom: 0;
+  // Reusing styles from .p-notification--information
+  // https://github.com/canonical-web-and-design/vanilla-framework/blob/main/scss/_patterns_notifications.scss#L190
+  &.note {
+    @include vf-highlight-bar($color-information, left, true);
+    @include vf-icon-info($color-mid-dark);
+  }
+
+  // Reusing styles from .p-notification--negative
+  // https://github.com/canonical-web-and-design/vanilla-framework/blob/main/scss/_patterns_notifications.scss#L181
+  &.danger {
+    @include vf-highlight-bar($color-negative, left, true);
+    @include vf-icon-error;
+  }
+
+  // Reusing styles from p-notification--caution
+  // https://github.com/canonical-web-and-design/vanilla-framework/blob/main/scss/_patterns_notifications.scss#L172
+  &.caution {
+    @include vf-highlight-bar($color-caution, left, true);
+    @include vf-icon-warning;
+  }
+
+  // Reusing styles from p-notification__title
+  // https://github.com/canonical-web-and-design/vanilla-framework/blob/main/scss/_patterns_notifications.scss#L81
+  .admonition-title {
+    font-size: 1rem;
+    font-weight: $font-weight-bold;
+    line-height: map-get($line-heights, default-text);
+    margin-bottom: 0.2rem !important;
```

Comment:
Why are these margins here hardcoded with some arbitrary values? Why is important needed? The p-notification-title seems to have margin set to 0.

Reply:
If I don't increase the lower margin it looks too close to the content: 

![Screenshot 2021-10-19 at 08 52 05](https://user-images.githubusercontent.com/58959073/137867020-81c13609-8ca9-41c2-9df5-c9559aed9b31.png)

I think it's because the title isn't an `<h5>` it's a `<p>` ? I've done some shimmying about and removed the `!important`, can you think of how I could go about this without the magic number?

Also, because the title and text aren't wrapped in the `p-notification__content` div, if the margin-top is set to 0 then it looks like this: 

![Screenshot 2021-10-19 at 08 55 30](https://user-images.githubusercontent.com/58959073/137867480-367c1711-add7-4018-87d0-67b578459790.png)

I've updated it trying to combine the styles needed from both `p-notification__content` and `p-notification__title` - let me know what you think.